app/code/Magento/Directory/view/adminhtml/templates/js/optional_zip_countries.phtml

```diff
@@ -12,6 +12,7 @@
 
 /**
  * @var \Magento\Backend\Block\Template $block
+ * @var \Magento\Framework\Escaper $escaper
  * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
```

Comment:
please combine all phpdoc blocks to a single one and remove not needed closing and opening PHP tags

Reply:
fixed